.changeset/sweet-meals-pay.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nordeck/matrix-neoboard-react-sdk': patch
+'@nordeck/matrix-neoboard-widget': patch
+---
+
+Change rounded corner rectngle element postion in the toolbar
```

Comment:
sorry, but since this is going into the changelog I think it is worth fixing

```suggestion
Change rounded corner rectangle element position in the toolbar
```